packages/template-block/src/hooks/useTemplateBlockData.ts

```diff
@@ -158,11 +158,16 @@ export const useTemplateBlockData = (appBridge: BlockProps['appBridge']) => {
     };
 };
 
-const getContentClasses = (textPositioning: TextPositioningType, textAnchoringVertical: AnchoringType): string => {
-    const alignContentItems = [TextPositioningType.Right, TextPositioningType.Left].includes(textPositioning)
-        ? verticalAlignmentToItemAlign[textAnchoringVertical]
-        : '';
-    return `tw-flex tw-gap-x-8 tw-gap-y-4 ${textPositioningToContentFlexDirection[textPositioning]} ${alignContentItems}`;
+const getContentClasses = (
+    hasPreview: boolean,
+    textPositioning: TextPositioningType,
+    textAnchoringVertical: AnchoringType
+): string => {
+    const alignContentItems =
+        hasPreview && [TextPositioningType.Right, TextPositioningType.Left].includes(textPositioning)
+            ? verticalAlignmentToItemAlign[textAnchoringVertical]
+            : '';
+    return `tw-flex tw-gap-x-6 tw-gap-y-4 ${textPositioningToContentFlexDirection[textPositioning]} ${alignContentItems}`;
```

Comment:
Here, I just added the `hasPreview` parameter, and added that to the `alignContentItems` condition (it's a bit hard to see). Also, I change `tw-gap-x-8` to `tw-gap-x-6`.